frontend/src/scenes/annotations/Annotations.tsx

```diff
@@ -83,7 +84,27 @@ export function Annotations(): JSX.Element {
             },
             sorter: (a, b) => annotationScopeToLevel[a.scope] - annotationScopeToLevel[b.scope],
         },
-        createdByColumn() as LemonTableColumn<AnnotationType, keyof AnnotationType | undefined>,
+        {
+            title: 'Created by',
+            dataIndex: 'created_by',
+            render: function Render(_: any, item) {
+                const { created_by, creation_type } = item
+                return (
+                    <div className="flex flex-row items-center">
+                        <ProfilePicture
+                            name={creation_type === 'GIT' ? 'GitHub automation' : created_by?.first_name}
```

Comment:
Since we're already committing ourselves to GitHub:
```suggestion
                            name={creation_type === 'GIT' ? 'GitHub Actions' : created_by?.first_name}
```
Though if we foresee this being used with GitLab etc. then "Git" would be more universal